### ISSUE TITLE
Remove "insecure" package

### ISSUE THREAD

**Issue by Dandandan** [waartaa/waartaa#72]
Is there a reason it is still included?

The insecure package defaults to read/write access to collections with no rules.

The [Meteor documentation](http://docs.meteor.com/#dataandsecurity) also recommends removing the package.

**Comment by rtnpro** [comment]
Yes, we need to remove it. Please feel free to drop a patch :) Thanks for
reporting it.
